bitnami/etcd/templates/cronjob.yaml

```diff
@@ -76,6 +76,8 @@ spec:
                   value: "3"
                 - name: ETCD_ON_K8S
                   value: "yes"
+                - name: MY_STS_NAME
```

Comment:
Could you add more context to the description or where does this variable come from please?